tests/sharingTest.js

```diff
@@ -848,12 +852,7 @@ describe('Main: Currently testing file/folder sharing,', function () {
         await capabilitiesGETRequestValidAuth(provider)
         await GETRequestToCloudUserEndpoint(provider)
 
-        // TODO: uncomment this line once the issue is fixed
-        // https://github.com/pact-foundation/pact-js/issues/577
-        // await Promise.all(testFiles.map(file => sharePOSTRequestWithExpirationDateSet(provider, file)))
-
-        await sharePOSTRequestWithExpirationDateSet(provider, testFiles[0])
-
+        await Promise.all(config.testFilesPath.map(file => sharePOSTRequestWithExpirationDateSet(provider, file)))
```

Comment:
`testFilesPath` contains the encoded file list, that matches what the consumer sends